garage/tf/algos/npo.py

```diff
@@ -358,22 +363,37 @@ def _build_entropy_term(self, i):
                     i.obs_var,
                     i.policy_state_info_vars,
                     name="policy_dist_info")
+                policy_log_likeli_flat = self.policy.distribution.log_likelihood_sym(  # noqa: E501
+                    i.action_var,
+                    policy_dist_info_flat,
+                    name="policy_log_likeli")
             else:
                 policy_dist_info_flat = self.policy.dist_info_sym(
                     i.flat.obs_var,
                     i.flat.policy_state_info_vars,
                     name="policy_dist_info_flat")
+                policy_log_likeli_flat = self.policy.distribution.log_likelihood_sym(  # noqa: E501
+                    i.flat.action_var,
+                    policy_dist_info_flat,
+                    name="policy_log_likeli")
+
+            if self._use_logli_entropy:
+                policy_entropy_flat = policy_log_likeli_flat
```

Comment:
@ryanjulian Shouldn't this be negative log likelihood if using it for entropy?

Reply:
Are you suggesting to change the name? 
log_likelihood_sym returns a negative log likelihood.

Reply:
Oh ok. Then just change the name. I did not notice that it returns negative log likelihood already.

Reply:
Done.